spec/hyacinth/adapters/digital_object_search_adapter/solr/document_generator_spec.rb

```diff
@@ -92,6 +100,33 @@
         expect(document['df_isbn_value_ssim']).to match_array isbn_values
       end
 
+      it 'does not index the value of the note field because it is of type textarea' do
+        expect(document['df_note_value_ssim']).to eq(nil)
+      end
+
+      it 'indexes presence of all non-textarea fields' do
+        expect(document['df_title_nonSortPortion_present_bi']).to eq(true)
+        expect(document['df_title_sortPortion_present_bi']).to eq(true)
+        expect(document['df_name_term_present_bi']).to eq(true)
+        expect(document['df_name_role_term_present_bi']).to eq(true)
+        expect(document['df_alternateTitle_value_present_bi']).to eq(true)
+        expect(document['df_isbn_value_present_bi']).to eq(true)
+      end
+
+      it 'indexes the presence of all textarea fields' do
+        expect(document['df_note_value_present_bi']).to eq(true)
+      end
+
+      it 'only adds presence keys for non-blank fields' do
+        expect(document.keys.select { |key| key.match?(/df_.+_present_bi/) }).to match_array(
+          [
+            'df_title_nonSortPortion_present_bi', 'df_title_sortPortion_present_bi', 'df_name_term_present_bi',
```

Comment:
Please remove the df_title references from this spec

Reply:
Will do post-HYACINTH-463-merge so that the tests still pass.  I'll rebase this branch on main after the merge is complete.